cmd/ethrex/l2/options.rs

```diff
@@ -22,21 +26,37 @@ pub struct Options {
         help_heading = "L2 options"
     )]
     pub sponsorable_addresses_file_path: Option<String>,
-    #[arg(long, value_parser = utils::parse_private_key, env = "SPONSOR_PRIVATE_KEY", help = "The private key of ethrex L2 transactions sponsor.", help_heading = "L2 options")]
-    pub sponsor_private_key: Option<SecretKey>,
+    #[arg(long, default_value = "0xffd790338a2798b648806fc8635ac7bf14af15425fed0c8f25bcc5febaa9b192", value_parser = utils::parse_private_key, env = "SPONSOR_PRIVATE_KEY", help = "The private key of ethrex L2 transactions sponsor.", help_heading = "L2 options")]
+    pub sponsor_private_key: SecretKey,
```

Comment:
Shouldn't this be optional?

Reply:
I tried to set it as optional, but that would leave an inconsistent state in another part. I'd rather leave the correct fix for this to another PR.